### ISSUE TITLE
Uncaught TypeError: Cannot read property 'toggle' of null

### ISSUE THREAD

**Issue by brbrr** [jeremyramin/terminal-plus#57]
[Enter steps to reproduce below:]

Right after i've update to 0.12.2 and statusBar appear on the bottom, i've tried to open terminal window by ctrl-` and got that issue. app restart fix that issue.

**Atom Version**: 1.0.19
**System**: Mac OS X 10.10.5
**Thrown From**: [terminal-plus](https://github.com/jeremyramin/terminal-plus) package, v0.12.2

### Stack Trace

Uncaught TypeError: Cannot read property 'toggle' of null

```
At /Users/brbrr/.atom/packages/terminal-plus/lib/status-bar.coffee:223

TypeError: Cannot read property 'toggle' of null
  at StatusBar.module.exports.StatusBar.toggle (/Users/brbrr/.atom/packages/terminal-plus/lib/status-bar.coffee:223:20)
  at atom-workspace.subscriptions.add.atom.commands.add.terminal-plus:toggle (/Users/brbrr/.atom/packages/terminal-plus/lib/status-bar.coffee:25:35)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:245:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:184:20)

```

### Commands

```
     -0:36.7.0 fuzzy-finder:toggle-file-finder (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-base16-tomorrow-dark-theme.theme-atom-dark-ui)
     -0:32.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -0:32.6.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-base16-tomorrow-dark-theme.theme-atom-dark-ui)
     -0:06.0 terminal-plus:toggle (div.settings-view.pane-item)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "goto",
      "ctags-status",
      "ruby-debugger",
      "rspec",
      "ruby-test",
      "script",
      "data-atom",
      "quick-query",
      "refactor",
      "symbols-view"
    ],
    "themes": [
      "atom-dark-ui",
      "base16-tomorrow-dark-theme"
    ]
  },
  "terminal-plus": {
    "core": {
      "scrollback": 2000
    },
    "style": {
      "theme": "pro",
      "fontFamily": "Monaco",
      "fontSize": 13,
      "animationSpeed": 2
    }
  }
}
```

### Installed Packages

```coffee
# User
advanced-open-file, v0.11.0
atom-beautify, v0.28.15
atom-ctags, v4.3.0
autocomplete-ctags, v0.3.2
autocomplete-ruby, v0.1.0
language-rspec, v0.3.0
linter, v1.9.1
linter-rubocop, v0.4.4
linter-ruby, v1.2.0
minimap, v4.15.0
minimap-find-and-replace, v4.3.1
minimap-git-diff, v4.1.8
rspec-snippets, v0.4.0
ruby-block, v0.3.5
synced-sidebar, v0.4.2
terminal-plus, v0.12.2
tool-bar, v0.1.9
tool-bar-main, v0.0.8

# Dev
No dev packages
```

**Comment by jeremyramin** [comment]
Thanks for reporting this. It should be fine now after the restart, but I do see what happened. I'll make some minor adjustments for this.

**Comment by jeremyramin** [comment]
Fixed as of v0.12.3
